contents/tutorials/intercom-session-replays.md

```diff
@@ -0,0 +1,238 @@
+    ui_host: 'https://us.posthog.com' // remove if EU
+ })
+}
+
+export function PHProvider({ children }) {
+  return <PostHogProvider client={posthog}>{children}</PostHogProvider>
+}
+```
+
+Once created, you can import `PHProvider` into your `layout.js` file and wrap your app in it:
+
+```js
+import "./globals.css";
+import { PHProvider } from './providers'
+
+export default function RootLayout({ children }) {
+  return (
+    <html lang="en">
+      <PHProvider>
+        <body>{children}</body>
+      </PHProvider>
+    </html>
+  );
+}
+```
+
+## 3. Add Intercom
+
+Start by going to your [installation settings page](https://app.intercom.com/a/apps/_/settings/web). Once there, enable "Install Messenger on your site or web app," choose the "Install with code snippet" tab, and then choose "Single-page apps." 
+
+To install Intercom in the `app/page.js` file, we:
+
+1. Set `APP_ID` as a variable, but you can use the pre-filled URL Intercom provides as well. 
+2. Add the first script to a `useEffect` to run on load.
+3. Add the `boot` call to the `handleSubmit` function where the email is available.
+
+Altogether, this looks like this:
+
+```js
+// app/page.js
+"use client"
+import { useEffect, useState } from 'react';
+
+export default function Home() {
+	const [email, setEmail] = useState('');
+  const APP_ID = "q0wg4mi9"; // replace this
+
+  useEffect(() => {
+    (function(){var w=window;var ic=w.Intercom;if(typeof ic==="function"){ic('reattach_activator');ic('update',w.intercomSettings);}else{var d=document;var i=function(){i.c(arguments);};i.q=[];i.c=function(args){i.q.push(args);};w.Intercom=i;var l=function(){var s=d.createElement('script');s.type='text/javascript';s.async=true;s.src='https://widget.intercom.io/widget/'+APP_ID;var x=d.getElementsByTagName('script')[0];x.parentNode.insertBefore(s,x);};if(document.readyState==='complete'){l();}else if(w.attachEvent){w.attachEvent('onload',l);}else{w.addEventListener('load',l,false);}}})();
+  }, []);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(e)
+    window.Intercom("boot", {
+      api_base: "https://api-iam.intercom.io",
+      app_id: APP_ID,
+      email: email
+    });
+  }
+
+  return (
+     <main>
+      <h1>Hello Intercom!</h1>
+      <h3>Enter your email:</h3>
+      <form onSubmit={handleSubmit}>
+        <input 
+          type="email"
+          value={email} 
+          onChange={(e) => setEmail(e.target.value)} 
+        />
+        <button type="submit">Submit</button>
+      </form>
+    </main>
+  );
+}
+```
+
+Now when you submit an email, Intercom launches with your email connected.
+
+![Intercom launched](../images/tutorials/intercom-session-replays/intercom.png)
+
+## 4. Adding session replays to Intercom
+
+The final piece is adding PostHog's session replays to Intercom. Since PostHog automatically captures session replay, all that is left to do is get the link and add it somewhere in Intercom. There are two spots we can add it.
```

Comment:
```suggestion
The final piece is adding PostHog's session replays to Intercom. Since PostHog automatically captures session replay, all that is left to do is get the link and add it in Intercom. There are two spots we can add it.
```